.github/workflows/docker-publish.yml

```diff
@@ -13,15 +13,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Export REACT_APP_CE_CURRENT_VERSION env var
+      - name: Export REACT_APP_APP_VERSION env var
         run: |
           git fetch --prune --unshallow
-          echo "REACT_APP_CE_CURRENT_VERSION=$(git describe --tags --abbrev=0) ($(git rev-parse --abbrev-ref HEAD):$(git log -1 --format=%h))" >> $GITHUB_ENV
+          echo "REACT_APP_APP_VERSION=$(git describe --tags --abbrev=0) ($(git rev-parse --abbrev-ref HEAD):$(git log -1 --format=%h))" >> $GITHUB_ENV
+      - name: Build npm package
```

Comment:
2 minutes for the build is *so* much better, especially given that it happens before deployment. Nice!